Modix.Bot/Modules/InfractionModule.cs

```diff
@@ -54,13 +56,13 @@ public InfractionModule(IModerationService moderationService, IUserService userS
                     new SortingCriteria { PropertyName = "CreateAction.Created", Direction = SortDirection.Descending }
                 });
 
-            if (infractions.Count == 0)
+            if (infractionResult.IsFailure || infractionResult.Result.Count == 0)
```

Comment:
I'm a little torn on this. Feels dishonest to say that there were no infractions if the service call failed. Would it make more sense to just say that we were unable to retrieve infraction information for the user?

Reply:
Agreed, forgot to fix this after I added the methods to ModixModule. Fixed.